### PR TITLE
game board IN and role view

### PR DESCRIPTION
**Pull request by CSimbulan**
Removed the temporary list of words.
Board gets generated now based on the match context state
Cards get a prop to determine their styling based on the roles from the match context, not sure if this entirely cheat proof.

Spymaster view:
image

Guesser view:
image

### CODE REVIEW

**Review by dproc96 — COMMENTED** [review]
This PR can't be tested without the match context, can you convert it to a draft for the time being?

**Comment by CSimbulan** [comment]
> 
> 
> This PR can't be tested without the match context, can you convert it to a draft for the time being?

Sure thing

**Comment by CSimbulan** [comment]
This can run now that the other assign roles socket got branched. Some changes made in the mean time:
- Assigning a role will send the player name and the player email through the socket to get stored in the game engine.
- If we use the name and 2 players have the same name it could give a player the wrong game board view, so email (or user's mongodb id) is better since it is unique.
- Still need to store the name as well to show them in the assign roles page (maybe there is a better way to do this?)

File: client/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME=du081ilw3
+REACT_APP_NEXT_PUBLIC_CLOUDINARY_UPLOAD_PRESET=fldfmkzr
```
Comment (dproc96) [inline client/.env]: You should ignore this env file

**Review by dproc96 — APPROVED** [review]
Looks good to me! Just double check the unused variables in the console. If they will be used in a future PR you can leave them as is, if not go ahead and remove them

**Pull request merged**